File: DevExpress-Demo/CS/MapDemo.Wpf/Controls/PhotoGallery/PhotoGalleryItemControl.cs
namespace MapDemo {
    public class PhotoGalleryItemControl : VisibleControl  {
        public PhotoGalleryItemControl() {
            DefaultStyleKey = typeof(PhotoGalleryItemControl);
            MouseLeftButtonUp += new System.Windows.Input.MouseButtonEventHandler(OnMouseButtonUp);
        }
        void OnMouseButtonUp(object sender, System.Windows.Input.MouseButtonEventArgs e) {
            e.Handled = true;
        }
    }
}

File: DevExpress-Demo/DevExpress.ProductsDemo.Wpf/VB/DevExpress.ProductsDemo.Wpf/Controls/PhotosGallery/PhotoGalleryItemControl.vb
Namespace ProductsDemo

    Public Class PhotoGalleryItemControl
        Inherits VisibleControl

        Public Sub New()
            DefaultStyleKey = GetType(PhotoGalleryItemControl)
            AddHandler MouseLeftButtonUp, New System.Windows.Input.MouseButtonEventHandler(AddressOf OnMouseButtonUp)
        End Sub

        Private Sub OnMouseButtonUp(ByVal sender As Object, ByVal e As System.Windows.Input.MouseButtonEventArgs)
            e.Handled = True
        End Sub
    End Class
End Namespace
